modules/openapi-generator/src/main/resources/scala-sttp/project/build.properties.mustache

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.4
```

Comment:
This file/version might be the cause of CI build failed with next error:

https://app.circleci.com/pipelines/github/OpenAPITools/openapi-generator/3736/workflows/1d16cd64-dd72-4f2b-8a68-92adfbdb29b7/jobs/17104/steps

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scala-sbt#compiler-bridge_2.13;1.2.3: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[error] ## Exception when compiling 14 sources to /home/circleci/OpenAPITools/openapi-generator/samples/client/petstore/scala-sttp/target/scala-2.13/classes
[error] The compiler bridge sources org.scala-sbt:compiler-bridge_2.13:1.2.3:compile could not be retrieved.
[error]  
[error] 	Note: Unresolved dependencies path:
[error] 		org.scala-sbt:compiler-bridge_2.13:1.2.3
[error] 		  +- org.scala-sbt.temp:temp-module-52f238aede5a1d8d7a484c2bec27a144810e2ac0:1.2.3
 ``` 
Could you try sbt.version=1.2.8? Should we even specify this version explicitly?

Reply:
Yes, the sbt version should be specified in `project/build.properties` so that the version is fixed and not platform-dependent. However, we should probably use `1.3.12`, which is the newest one.